enterprise/frontend/src/metabase-enterprise/embedding_iframe_sdk/embed.v1.unit.spec.ts

```diff
@@ -0,0 +1,70 @@
+import { MetabaseEmbed } from "./embed.v1";
+
+describe("embed.v1.js script tag for sdk iframe embedding", () => {
+  const defaultSettings = {
+    apiKey: "test-api-key",
+    instanceUrl: "http://localhost:3000",
+    target: "#embed-container",
+  };
+
+  it("throws when target element is not found", () => {
+    expect(() => {
+      new MetabaseEmbed({ ...defaultSettings, target: "#not-existent-target" });
+    }).toThrow('cannot find embed container "#not-existent-target"');
+  });
+
+  it("throws when target element is undefined", () => {
+    expect(() => {
+      // @ts-expect-error -- we are testing for incorrect configuration
+      new MetabaseEmbed({ ...defaultSettings, target: undefined });
+    }).toThrow('cannot find embed container "undefined"');
```

Comment:
I kind of expected a different error similar to other missing property errors like `target must be provided`

Reply:
true, I must've forgot to add this in. lemme add that real quick